### ISSUE TITLE
Something's wrong with the houses

### ISSUE THREAD

**Issue by Gurumi-Heist** [Armoury-Software/ArmouryFiveM#22]
I've just done a fresh install of the server by following the instructions on the github page. I've got an error with the house the movement I've done registering. 
![image](https://user-images.githubusercontent.com/100353827/184714729-6ff7a91e-6e2e-457e-848c-31bf34e750f1.png)

**Comment by maziluradu** [comment]
Indeed, the error appears, but it shouldn't disrupt any of the house system behaviour. I'll come up with a fix to include it in the next release. Thank you for issuing the bug!
